jumpscale/packages/auth/bottle/auth.py

```diff
@@ -174,7 +174,7 @@ def access_denied():
         Renders access denied page
     """
     email = request.environ.get("beaker.session").get("email")
-    next_url = request.query.get("next_url")
+    next_url = request.query.get("next_url", "/")
```

Comment:
shouldn't it be the logout url so he can login again?
```suggestion
    next_url = request.query.get("next_url", "/auth/logout")
```